### DISCUSSION TITLE
问题：最近无法使用

### DISCUSSION THREAD

**Discussion by zjhzsz** [shadowsocks/shadowsocks-android#2834] [Q&A]
最近发生，可以链接上服务器，握手延时也很短，握手时有些流量，其余无流量，无上传下载量。
未改动过设置，怀疑服务器ip地址被封，更换了ip地址问题依旧。
不知道哪里出问题，请帮忙解决，谢谢。

**Comment by zjhzsz** [comment]
![Screenshot_20211125-053828_Shadowsocks](https://user-images.githubusercontent.com/22661380/143316141-426c2210-d7b6-48f7-8202-8608c852a318.jpg)
![Screenshot_20211125-053733_Shadowsocks](https://user-images.githubusercontent.com/22661380/143316155-aeef0001-b99a-4cd2-b670-e2dd4727cc7b.jpg)

**Reply by zjhzsz** [reply]
大家没有类似情况吗？我用vultr ，国内电信，联通网络。

**Reply by dev4u** [reply]
你有点误解，测试握手延时的链接，跟代理不一定有关系，要视乎你用的代理模式。

**Reply by zjhzsz** [reply]
> 

我没有改动任何设置，原来能用，突然一天就不能用了

**Reply by ermeiyao** [reply]
我也遇到了，我使用的是gost代理https,pc端使用正常，就是在手机上无法翻墙
